cli/ballerina-cli/src/test/resources/test-resources/diagnostics-test-files/bal-project-error/project2/main.bal

```diff
@@ -0,0 +1,18 @@
+import ballerina/http;
```

Comment:
don't use library functions when writing tests. In this case write a Listener class and use it.